docs/doxygen-user/multi-user/installSolr.dox

```diff
@@ -0,0 +1,145 @@
+/*! \page install_solr Install and Configure Solr
+
+\section install_solr_overview Overview
+
+Autopsy uses Apache Solr to store keyword text indexes.  A central server is needed in a multi-user cluster to maintain and search the indexes.  
+
+A new Solr core is created for each case and the core/index is stored in the case folder (not on the local drive of the Solr server). 
+
+Solr's embedded Zookeeper is also used as a coordination service for Autopsy. 
+
+\section install_solr_prereq Prerequisites
+
+We use Bitnami Solr, which packages Solr as a Windows service. 
+
+You will need:
+- A 64-bit version of the Java Runtime Environment (JRE) from http://www.oracle.com/technetwork/java/javase/downloads/jre8-downloads-2133155.html. 
+- The Apache Solr 4.10.3-0 installation package.  This is no longer available from its original source, but you can find it on our site: https://sourceforge.net/projects/autopsy/files/CollaborativeServices/Solr.
```

Comment:
Based on a recent forum question, I might add a note that you must get this version of solr and can't use a newer version.

Reply:
Added.